### ISSUE TITLE
Fresh install of v1.17.0-rc.3-k3s.1 with HA fails with cloud-controller-manager permission issue

### ISSUE THREAD

**Issue by davidnuzik** [rancher/k3s#1229]
**Version:**
v1.17.0-rc3-k3s.1 (fresh install with HA)

**Describe the bug**
With HA or otherwise multi-node a fresh install of v1.17 rc2 and rc3 fails. All four pods will stay stuck in a "Pending" state (coredns, helm-install-traefik, local-path-provisioner, metrics-server).
When we review the logs we see:

```
Dec 20 17:39:42 ip-172-31-20-233 k3s[6167]: E1220 17:39:42.820482    6167 leaderelection.go:331] error retrieving resource lock kube-system/cloud-controller-manager: leases.coordination.k8s.io "cloud-controller-manager" is forbidden: User 
Dec 20 17:39:44 ip-172-31-20-233 k3s[6167]: W1220 17:39:44.090813    6167 garbagecollector.go:639] failed to discover some groups: map[metrics.k8s.io/v1beta1:the server is currently unable to handle the request]
Dec 20 17:39:46 ip-172-31-20-233 k3s[6167]: E1220 17:39:46.439101    6167 leaderelection.go:331] error retrieving resource lock kube-system/cloud-controller-manager: leases.coordination.k8s.io "cloud-controller-manager" is forbidden: User 
Dec 20 17:39:50 ip-172-31-20-233 k3s[6167]: E1220 17:39:50.172519    6167 leaderelection.go:331] error retrieving resource lock kube-system/cloud-controller-manager: leases.coordination.k8s.io "cloud-controller-manager" is forbidden: User 
Dec 20 17:39:52 ip-172-31-20-233 k3s[6167]: E1220 17:39:52.344853    6167 leaderelection.go:331] error retrieving resource lock kube-system/cloud-controller-manager: leases.coordination.k8s.io "cloud-controller-manager" is forbidden: User 
```

**To Reproduce**
Set up an HA environment with a fresh install of v1.17 k3s (rc2 or rc3)
Observe the pods will never become ready
Review the logs and notice the cloud-controller-manager permission issue

**Expected behavior**
All pods ready and K3s cluster functioning normally.

**Actual behavior**
Pods stuck in Pending state.

**Additional context**
Should be an easy permission fix.

**Comment by ShylajaDevadiga** [comment]
Fixed in v1.17.0-rc4-k3s.1

**Comment by davidnuzik** [comment]
Should be fixed in rc4 -- needs testing and needs PR attached.
